.gitignore

```diff
@@ -59,3 +59,6 @@ spec/system/ucas_matching/matching_data_example.zip
 # jmeter stuff
 ruby-jmeter.jmx
 jmeter.log
+
+# RubyMine files
+.idea
```

Comment:
I'm not sure how you feel about this, so I've included it here for discussion. 

I spent some time trying to configure a global excludes file, as discussed in the comment on line 5 of the gitignore but I was unsuccessful. This is a relatively low impact solution and will mean that any other team members in future also won't have to worry about it or spend time on it.

Equally, I'm happy to remove this change and pair with someone to figure out how to get it set up correctly. And then update the docs for future devs.